### DISCUSSION TITLE
Multiple client sessions on Win 10

### DISCUSSION THREAD

**Discussion by jlguit** [jamulussoftware/jamulus#1159] [Technical Support & Help]
Start using Jamulus one month ago, love it, I’m trying to setup 2 or more client sessions on one computer win 10, to send audio on separate channels, problem I have is when I open the 2nd session the first one close. I read somewhere that you can do it with separate ini file, each session works fine alone. Any idea? 

Envoyé de mon iPad

**Comment by ann0see** [comment]
Hi @jlguit!

If I understand correctly, you‘d like to run two clients on Windows. First of all, I must admit that I don’t use Windows for audio/Jamulus, so I might not be much help. As far as I remember (not sure) ASIO only supports one application per device? 

Maybe voicemeeter banana or just using two different ASIO drivers might solve the problem. 

I don’t know anything about an ini file argument, but you could have a look at the Jamulus.ini file in %AppData%.

**Reply by dcorson-ticino-com** [reply]
What I don't understand is that you say the first instance is closed when you open the second one.
I see no reason why that should happen.
I have run several clients at a time for testing under Win10, no problems.
You can give each client a different ini file by using the command line option -i.
See the the command line doc here:
https://jamulus.io/wiki/Command-Line-Options
What ann0see said it true, however, most ASIO drivers do not like multiple applications.

**Reply by jlguit** [reply]
I test each client separately with different audio port with asio driver from Presonus. I use a VSL44 - 4 ports. Each works perfectly, when I open the second one, not even connect to the Server, after 15, 20 or 25 seconds, the first one close, If the first one is connected to the Server, it will disconnect. Somebody did qnswer me that Asio will not support 2 Sessions. I am still digging for making work, My next try will be with Asio for All,

Jean-Louis

**Reply by jlguit** [reply]
> Hi @jlguit!
> 
> If I understand correctly, you‘d like to run two clients on Windows. First of all, I must admit that I don’t use Windows for audio/Jamulus, so I might not be much help. As far as I remember (not sure) ASIO only supports one application per device?
> 
> Maybe voicemeeter banana or just using two different ASIO drivers might solve the problem.
> 
> I don’t know anything about an ini file argument, but you could have a look at the Jamulus.ini file in %AppData%.

Can you tell me which audio device you were using to make it work?

**Reply by ann0see** [reply]
As I said, I don't use Windows for audio stuff, therefore I won't be much help. 

If I understand correctly, Jamulus closes itself once you open two clients? You don't get any error message? Do you?

I would try Voicemeeter banana or JACK.

**Comment by gamfen** [comment]
I'm having the same issue. I'd like to use Jamulus with two clients, so I can have 2 microphones on seperate channels for the other musicians to mix seperately.
Problem is, as soon as I run the second client, it stops sending any audio signal via the first client and I can only use the second client/microphone.
I'm using the Behringer UMC204HD with its native ASIO on Win 10. Any ideas how to fix this?
